playbooks/set-osd-global-config.yml

```diff
@@ -0,0 +1,20 @@
+- name: set osd_pool_default_size
+  hosts: rhcs5-admin
```

Comment:
Is this host "rhcs5-admin" intended?
I don't see hosts with this label added to the conf file.